src/main/java/taxi/controller/driver/AddDriverController.java

```diff
@@ -21,11 +21,19 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
     }
 
     @Override
-    public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
-        String name = req.getParameter("name");
-        String licenseNumber = req.getParameter("license_number");
-        Driver driver = new Driver(name, licenseNumber);
-        driverService.create(driver);
-        resp.sendRedirect(req.getContextPath() + "/drivers/add");
+    public void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException, ServletException {
+        try {
+            String name = req.getParameter("name");
+            String licenseNumber = req.getParameter("license_number");
+            String login = req.getParameter("login");
+            String password = req.getParameter("password");
+            Driver driver = new Driver(login, password, name, licenseNumber);
+            driverService.create(driver);
+            resp.sendRedirect(req.getContextPath() + "/index");
```

Comment:
```suggestion
            resp.sendRedirect(req.getContextPath() + "/login");
```

Reply:
Why? If driver already login and want to add new driver

Reply:
imagine registration in gmail or instagram
you should register yourself, not other people
and after registration you probably want to login